lighthouse-core/lib/traces/pwmetrics-events.js

```diff
@@ -37,64 +36,64 @@ class Metrics {
         getTs: auditResults => {
           const fmpExt = auditResults['first-meaningful-paint'].extendedInfo;
           return fmpExt.value.timestamps.navStart;
-        }
+        },
```

Comment:
this is going to get old :P does it do the whole file when you change anything in it and save?